packages/ontario-frontend-cli/create/templates/package.njk

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "rm -rf dist && eleventy",
     "debug": "rm -rf dist && DEBUG=* eleventy",
-    "serve": "eleventy --serve"
+    "serve": "eleventy --serve",
+    {% for scriptName, scriptValue in addScripts %}"{{ scriptName }}": "{{ scriptValue | safe }}"{% endfor %}
```

Comment:
This feels a little complicated to me, do we need to determine and pass scripts this way? Or could it just be a condition? It shouldn't need to be custom/dynamic per project in this sense